### PR TITLE
introduce new module session_with_id for session definition using `session_id`

### PR DESCRIPTION
**Pull request by ttngu207**

### CODE REVIEW

**Comment by CBroz1** [comment]
Summarizing: `subject.py` was changed to `subject_with_datetime.py`, and the `__init__.py` includes a line to make sure this is called as the default. Alternatively, someone could call `subject_with_id.py` which adds and int ID primary key (keeping datetime as a secondary key) to the session table, but is otherwise the same.

```python
class Session(dj.Manual):
    definition = """
    -> Subject
    session_id: int
    ---
    session_datetime: datetime
    """
```

**Comment by ttngu207** [comment]
> Summarizing: `subject.py` was changed to `subject_with_datetime.py`, and the `__init__.py` includes a line to make sure this is called as the default. Alternatively, someone could call `subject_with_id.py` which adds and int ID primary key (keeping datetime as a secondary key) to the session table, but is otherwise the same.
> 
> ```python
> class Session(dj.Manual):
>     definition = """
>     -> Subject
>     session_id: int
>     ---
>     session_datetime: datetime
>     """
> ```

@CBroz1, I don't quite get your comment above. Why is `Subject` relevant here? Why do we need a notion of `subject_with_id` or `subject_with_datetime`?

**Comment by CBroz1** [comment]
edited to fix: `subject.py` -> `session.py`

**Comment by ttngu207** [comment]
@CBroz1 , nice, thanks for the summary. I think it's ready to be merged.

**Comment by tdincer** [comment]
This enhancement is urgently required for one project. Could we complete the merge today?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [unreleased] 
+### Added
++ SessionExperimenter
++ SessionNote
++ NWB export
```
Comment (kabilar) [inline CHANGELOG.md:9]:
```suggestion
+ Module (`session_with_id`) with session primary key as integer
```

**Review by kabilar — APPROVED** [review]
Thanks @ttngu207.  Looks great.  A couple of minor comments above.

**Comment by tdincer** [comment]
We need this enhancement immediately.

**Comment by ttngu207** [comment]
@kabilar done. It is ready to be merged.

**Review by kabilar — APPROVED** [review]
Thanks @ttngu207!

**Pull request merged**